frontend-react/src/components/Content/Templates/IATemplate.tsx

```diff
@@ -0,0 +1,68 @@
+import { useCallback } from "react";
+
+import { ContentDirectory } from "../MarkdownDirectory";
+
+import { IACardGridTemplate } from "./IACardGridTemplate";
+import IASideNavTemplate from "./IASideNavTemplate";
+import { IAMeta, IARouter } from "./IAMeta";
+
+/** Template names! Add the universal template key here whenever
+ * you make a new template. */
+export enum TemplateName {
+    CARD_GRID = "card-grid",
+    SIDE_NAV = "side-nav",
```

Comment:
I read this template name and the component name `IASideNavTemplate` to mean `template that will render a side nav` when it seems to be more like `side nav with content`. I'm not sure how exactly to make this clearer, since `IASideNavWithContent` isn't great, but open to ideas... If nothing comes up this is fine, just want to make it as easy to understand as possible